meetings/2022-02/registration.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: meeting
+title: Registration
+meeting: true
+meeting_nr: 202109
+---
+
+Registration is closed now.
```

Comment:
Is the plan to ever open it? If not, just delete this page?

Reply:
Sorry about that. The plan was to have a smallish closed meeting (in part due to the pandemic). I must have forgotten to delete the registration.md after copying it from the summer school folder. I'll fix it now.